core/raydp-main/src/main/java/org/apache/spark/raydp/SparkOnRayConfigs.java

```diff
@@ -88,4 +88,7 @@ public class SparkOnRayConfigs {
      * only.
      */
     public static final String RAY_PREFER_CLASSPATH = "spark.ray.preferClassPath";
+
+    public static final String RAYDP_LOGFILE_PREFIX_CFG =
```

Comment:
can you please add comment why we set this file prefix?

Reply:
done

Reply:
can you fix the style to pass the CI？

Reply:
just fixed